### DISCUSSION TITLE
No "RWY" on FMA

### DISCUSSION THREAD

**Discussion by andrejetcat** [flybywiresim/aircraft#4933] [Q&A] [answered]
hello, 

I do not get the "RWY" Indication on Takeoff-run on the FMA even when i select the LOC/ILS frequency manunally.

I remember it works months ago. 

Any Ideas or solutions, or just not jet implemented?

**Comment by aguther** [comment, marked as answer]
I guess you're talking about development version?

This version is more exact in the conditions to engage RWY, there are several conditions that must apply:
- flight directors active
- valid V2 entered
- not in CONF 0
- more than 30s on ground
- throttles >= FLX/MCT
- LOC valid and is received
- LOC deviation less than 1/2 dot
- heading within 20° to LOC bearing
- SRS engaged more than 0.9 s

If all of the above are true, RWY will engage.

Keep in mind that MSFS has a lot of ILS issues one of them being that the LOC while in real-life being able to be received on the specific runway, it's often not the case in MSFS.

**Reply by andrejetcat** [reply]
thank you for your answer, 

for shure, all these conditions are apply! But NO RWY for me. I use latest DEV version.

**Reply by andrejetcat** [reply]
maybe an issue with my Thrustmaster Arbus Throttle/Flap/speedbrake lever?  Like the (old ans solved issue) with flaps 1 on approach? I tested it again and again, no RWY for me! All set like in real life in my real office A320

**Reply by aguther** [reply]
Can you post a screenshot on a runway in question with LS active so I can see if the LOC is received and the deviation from it? Does SRS engage correctly for you?

There are so many airports where LOC is incorrectly unavailable on the runway…that‘s why I‘m asking…

**Reply by andrejetcat** [reply]
shure, i try to give you a proper screenshot later this day. SRS works as it should. At this point, another great "**thank you**" for all you are doing here for an awesome experience!!!

**Reply by andrejetcat** [reply]
as I feared, finally it works! This time I tested it @EDDK, RWY 14L with manually entered LS. RWY works fine at this Airport. 

...but, @EDDL and LEPA (my last 8 legs) both addon scenery, it doesen´t! Maybe as you said, a MSFS issuse with the LOC! 

Thanks again for the great support!

**Reply by aguther** [reply]
Thank you for the kind words!

You cannot imagine how all those ILS issues annoy me! One reason why this happens is when you get below the height of the antenna the signal is lost independent from other things like position or distance...
